src/target/llvm/codegen_llvm.cc

```diff
@@ -88,7 +88,7 @@ void CodeGenLLVM::InitTarget(llvm::TargetMachine* tm) {
       native_vector_bits_ = 128;
     } else {
       native_vector_bits_ = 128;
-      std::string arch_name = tm->getTargetTriple().getArchName();
+      std::string arch_name = std::string(tm->getTargetTriple().getArchName());
```

Comment:
`StringRef` has a function `str` that returns `std::string`.

Reply:
Good to know. We decided on using explicit conversion because it works for older version of llvm as well as llvm_10.